test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/base/PipelineBaseE2EIT.java

```diff
@@ -130,6 +131,27 @@ public PipelineBaseE2EIT(final PipelineTestParameter testParam) {
         pipelineWatcher = new PipelineWatcher(containerComposer);
     }
     
```

Comment:
1, `cleanUpDataSource` should be invoked in every unit test method.

2, NATIVE env type could be embed in `cleanUpDataSource`.